core/runtime/op_driver/erased_future.rs

```diff
@@ -1,20 +1,24 @@
 // Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+#![allow(dead_code)]
```

Comment:
Yikes, I don't like these because `clippy` doesn't catch that the code is actually used and they rot forever.

Reply:
Let me see what I can trim here. I might be able to clean it up.